.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "${workspaceFolder}/venv/bin/python"
```

Comment:
This looks like something that needs to be removed and added to the `.gitignore`

Reply:
yes, that needs to be removed